File: CourceObjective-C/DatePickerExample/DatePickerExample/ViewController.m
//
//  ViewController.m
//  DatePickerExample
//
//  Created by Vanilson NiLL on 09/07/2018.
//  Copyright © 2018 Vanilson NiLL. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)dateOnClick:(id)sender {
    NSDate *theDate = [_birthdayDatePicker date];
    NSDateFormatter *dateFormat = [[NSDateFormatter alloc] init];
    [dateFormat setDateFormat:@"d/M/yyyy HH:mm"];
    NSString *textDate = [[NSString alloc] initWithFormat: @"Data: %@", [dateFormat  stringFromDate:theDate]];
    _dateLabel.text = textDate;
    //[theDate release];
}


@end

File: App06CourseIOS/App06CourseIOS/ViewController.swift
//
//  ViewController.swift
//  App06CourseIOS
//
//  Created by Vanilson on 27/06/17.
//  Copyright © 2017 Vanilsonvl. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        if segue.identifier == "LauncherCoin" {
            let random = arc4random_uniform(2)
            let destinationViewController = segue.destination as! DetailViewController
            destinationViewController.numberReceive = Int(random)
        }
    }

}

